src/main/java/org/gridsuite/study/server/StudyService.java

```diff
@@ -343,6 +347,41 @@ public StudyService(
         });
     }
 
+    public Mono<Collection<String>> getExportFormats() {
+        String path = UriComponentsBuilder.fromPath(DELIMITER + NETWORK_CONVERSION_API_VERSION + "/export/formats")
+                .toUriString();
+
+        ParameterizedTypeReference<Collection<String>> typeRef = new ParameterizedTypeReference<Collection<String>>() { };
+
+        return webClient.get()
+                .uri(networkConversionServerBaseUri + path)
+                .retrieve()
+                .bodyToMono(typeRef);
+    }
+
+    public Mono<ExportNetworkInfos> exportNetwork(String studyName, String format) {
+        Mono<UUID> networkUuidMono = getStudyUuid(studyName);
+
+        Mono<byte[]> networkDataMono = networkUuidMono.flatMap(uuid -> {
+            String path = UriComponentsBuilder.fromPath(DELIMITER + NETWORK_CONVERSION_API_VERSION + "/networks/{networkUuid}/export/{format}")
+                    .buildAndExpand(uuid, format)
+                    .toUriString();
+
+            return webClient.get()
+                    .uri(networkConversionServerBaseUri + path)
+                    .retrieve()
+                    .bodyToMono(byte[].class);
```

Comment:
Why use bodyToMono(byte[].class) and then manually parsing with the objectmapper ? Instead, use bodyToMono(ExportNetworkInfos.class) ?